setup.py

```diff
@@ -7,7 +7,7 @@
 VERSION = open(os.path.join(here, 'VERSION')).read()
 
 install_requires = [
-    'pyramid',
+    'pyramid==1.6.1',
```

Comment:
Is this absolutely necessary? I think we should leave it up to the api developer to choose which version it wants to use. If there is a minimum version required, it should be specified as `pyramid=>x.y.z`